site/docs/view-spec.md

```diff
@@ -0,0 +1,256 @@
+| Optional | session-namespace | An array of strings indicating namespace at the time view was created / replaced. Used similar to ‘session-catalog’ above. |
+| Optional | field-aliases | A list of strings of field aliases E.g. a list of alias_name info specified in the following create view statement. `CREATE VIEW v (alias_name COMMENT 'docs', alias_name2, ...) AS SELECT ...` |
+| Optional | field-docs | A list of strings of field comments E.g. a list of ‘comment’ info specified in the following create view statement. `CREATE VIEW v (alias_name COMMENT 'docs', alias_name2, ...) AS SELECT ...` |
+
+“version-log” is an array of structs describing the log of the versions created. The struct has the following fields:
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | timestamp-ms | Timestamp expressed as ms since epoch for when the version of the view indicated in version-id field below was created |
+| Required | version-id | Version id of the view |
+
+## Appendix A: An Example
+
+The JSON metadata file format is described using an example below.
+
+Imagine the following sequence of operations:
+
+* `CREATE TABLE base_tab(c1 int, c2 varchar);`
+* `INSERT INTO base_tab VALUES (1,’one’), (2,’two’);`
+* `CREATE VIEW common_view AS SELECT * FROM base_tab;`
+* `CREATE OR REPLACE VIEW common_view AS SELECT count(*) AS my_cnt FROM base_tab;`
+
+The metadata json file created at the end of step 3 looks as follows. The file path looks like:
+`s3://my_company/my/warehouse/anorwood.db/common_view`
+
+The path is intentionally similar to the path for iceberg tables and contains a ‘metadata’ directory. (`METASTORE_WAREHOUSE_DIR/<dbname>.db/<viewname>/metadata`)
+
+The metadata directory contains View Version Metadata files. The text after '=>' symbols describes the fields.
+```
+{
+  "format-version" : 1, =>Json format. Will change as format evolves.
+  "location" : "s3n://my_company/my/warehouse/anorwood.db/common_view",
+  "current-version-id" : 1, => current / latest version of the view. ‘1’ here since this metadata was created when the view was created.
+  "properties" : {  => shows properties of the view
+    "comment" : "View captures all the data from the table" => View comment
+  },
+  "versions" : [ { => Last few versions of the view. Controlled by the table property: “version.history.num_entries”
+    "version-id" : 1,
+    "parent-version-id" : -1,
+    "timestamp-ms" : 1573518431292,
+    "summary" : {
+      "operation" : "create", => View operation that caused this metadata to be created
+    	"engineVersion" : "presto-350",=> Version of the engine that performed the operation (create / replace)
+    },
+    "representations" : [ { => SQL metadata of the view
+      "type" : "sql",
+      "sql" : "SELECT *\nFROM\n  base_tab\n", => original view SQL
+      "schema" : { => Schema of the view expressed in Iceberg types
```

Comment:
Shouldn't this be in `schemas` and referenced by ID either here or in versions?

Reply:
You are right